src/wallet/db.h

```diff
@@ -166,6 +166,42 @@ class BerkeleyDatabase
     bool IsDummy() { return env == nullptr; }
 };
 
+namespace {
+    /** RAII class that provides access to a DB_DBT_MALLOC Dbt */
+    class MallocedDbt {
+    private:
```

Comment:
nit: You can drop the `private` access modifier. `m_dbt` is implicitly declared private, because private is the default class access level.